components/removed_from_channel_modal/index.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/common';
+
+import {closeModal} from 'actions/views/modals';
+
+import RemovedFromChannelModal from './removed_from_channel_modal';
+
+const mapStateToProps = createSelector(
```

Comment:
Why are you using a selector here instead of just a plain function? Usually, we'd just do this as
```javascript
function mapStateToProps(state) {
    return {
        currentUser: getCurrentUser(state),
    };
}
```

Reply:
Oh I had used another component's `index.js` as a reference (`edit_channel_purpose_modal`) originally, I'll be sure to change it accordingly